### DISCUSSION TITLE
PDFs do not respect the sort applied using a RefinableString

### DISCUSSION THREAD

**Discussion by steveh2023** [microsoft-search/pnp-modern-search#3506] [Q&A] [answered]
Hi,

I have set an a-z sort using the RefinableString method as title can not be sorted...

https://ibb.co/2SqXCDM

It works for word documents but not pdfs...

Please can you tell me how to make the a-z sort apply also to pdfs?

Thanks....

**Comment by kasperbolarsen** [comment]
Have you checked the values in the RefinableString for the PDF files? 

You might have to figure out which crawled property from the PDF that is similar to Title, without checking I would expect it to be related to FileName or something like that

**Reply by steveh2023** [reply]
How do you check the values in the RefinableString for pdfs?

Thanks.

**Comment by kasperbolarsen** [comment]
One option is to use the debug layout, or using one of the tools here: https://microsoft-search.github.io/pnp-modern-search/using-query-tools-to-verify-issues/

**Comment by kasperbolarsen** [comment, marked as answer]
I have this mapping on one of my tenants and PDF's are sortable by using RefinableString09 in this case:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/6a376455-08eb-4986-9e49-936a6c2c5694)

**Reply by steveh2023** [reply]
Thanks very much for the info - I will try that.

**Reply by steveh2023** [reply]
Oddly I do not have Mail:#5 available, I have the other 2 but not Mail:#5...the closest was: Mail:5

**Reply by steveh2023** [reply]
This does now work using basic:#10, office:#2 and mail:5 - there is no mail:#5 here...not sure why.
